nav2_amcl/include/nav2_amcl/amcl_node.hpp

```diff
@@ -210,6 +211,16 @@ class AmclNode : public nav2_util::LifecycleNode
     const std::shared_ptr<std_srvs::srv::Empty::Request> request,
     std::shared_ptr<std_srvs::srv::Empty::Response> response);
 
+  // service server for providing an initial pose guess
+  rclcpp::Service<nav2_msgs::srv::SetInitialPose>::SharedPtr initial_guess_srv_;
+  /*
+   * @brief Service callback for a global relocalization request
+   */
+  void initialGuessCallback(
```

Comment:
`initialPoseReceivedSrv`

Reply:
Renamed the callback name to initialPoseReceivedSrv.